cms/templates/howitworks.html

```diff
@@ -14,8 +16,8 @@
 <div class="wrapper-content-header wrapper">
   <section class="content content-header">
     <header>
-      <h1><span class="wrapper-text-welcome">${_("Welcome to {studio_name}").format(
-        studio_name=u'</span><span class="logo">{studio_name}</span>'.format(
+      <h1><span class="wrapper-text-welcome">${Text(_("Welcome to {studio_name}")).format(
+        studio_name=HTML(u'</span><span class="logo">{studio_name}</span>').format(
```

Comment:
Why are we using another span with logo class. Wouldn't that be easier if we just include `studio_name` as bellow :

```
<h1><span class="wrapper-text-welcome">${_("Welcome to {studio_name}").format(
          studio_name=settings.STUDIO_NAME
      )}</span></h1>
```
otherwise 

```
<h1><span class="wrapper-text-welcome">${Text(_("Welcome to {span_start}{studio_name}{span_end}")).format(
          span_start=HTML('<span class="logo">'),
          studio_name=settings.STUDIO_NAME,
          span_end=HTML('</span>')
      )}</span></h1>
```

This would need to tune logo class as 
```
.view-howitworks.not-signedin .content-header .logo {
    margin-left: -2px;
}
```
@robrap @cahrens what do you think ?